be/test/exprs/agg/aggregate_test.cpp

```diff
@@ -639,6 +639,34 @@ TEST_F(AggregateTest, test_stddev_samp) {
     }
 }
 
+TEST_F(AggregateTest, test_maxby) {
+    const AggregateFunction* func = get_aggregate_function("max_by", TYPE_VARCHAR, TYPE_INT, true);
```

Comment:
would you add some special cases, like columns with null, or const column?

Reply:
what do you mean by const column?

Reply:
> would you add some special cases, like columns with null, or const column?

case with null columns has been added, for const column, an error will be reported.
ERROR 1064 (HY000): max_by function args must be column

Reply:
you can refer `be/test/column/const_column_test.cpp`  to create const columns to tests `max_by`.

Reply:
ok